src/main/java/underdogs/devbie/question/domain/HashtagRepository.java

```diff
@@ -0,0 +1,12 @@
+package underdogs.devbie.question.domain;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface HashtagRepository extends JpaRepository<Hashtag, Long>  {
+
+    @Query("select h from Hashtag h where h.tagName.name = :tagName")
+    Optional<Hashtag> findByTagName(String tagName);
```

Comment:
요기서 계속 에러가떠서 디버깅해봤는데요, 저번에 코일이 겪었던같은 이슈인 것 같습니다. 
Repository에서 파라미터 tagName을 못찾는 것인데요.

```suggestion
    @Query("select h from Hashtag h where h.tagName.name = :tagName")
    Optional<Hashtag> findByTagName(@Param("tagName") String tagName);
```
이렇게 변경하면 잘 동작하네요.

Reply:
그때 통일하기로했는데, 제가 깜빡했네요 죄송합니다!!